### ISSUE TITLE
[BUG] Radarr reacquire deletes the MAIN movie when any extra/trailer/clip under the folder is flagged

### ISSUE THREAD

**Issue by tikibozo** [aetaric/checkrr#170]
### Version
3.6.1

### Summary
When checkrr flags **any** file located under a Radarr movie's folder — a trailer, featurette, `/other` clip, sample, or partial download — `Radarr.RemoveFile` deletes the movie's **main** `MovieFile` and triggers a re-search, even though the flagged file is *not* the movie file. This deletes good main films.

### Root cause
`connections/radarr.go`, `RemoveFile`:

```go
for _, movie := range movieList {
    if strings.Contains(r.translatePath(path), movie.Path) {   // matches the FOLDER contains the path
        movieID = movie.ID
        ...
        if movie.MovieFile.ID != 0 {
            r.server.DeleteMovieFiles(movie.MovieFile.ID)        // deletes the MAIN movie file regardless
            r.server.SendCommand(... "RefreshMovie" ...)
            r.server.SendCommand(... "MoviesSearch" ...)
            return true
        }
    }
}
```

The match is on the **movie folder containing the flagged path**, after which it unconditionally deletes `movie.MovieFile.ID` (the main film). It never verifies that the flagged path *is* the main movie file.

For contrast, `connections/sonarr.go` `RemoveFile` only deletes on an **exact** file match: `if file.Path == s.translatePath(path)`.

### Impact
Data loss on the main film. Triggers: a trailer/featurette/`/other` clip, a `*.f616.mp4` partial download, or a sample.

### Suggested fix
Mirror Sonarr's exact-match: `if movie.MovieFile.ID != 0 && movie.MovieFile.Path == r.translatePath(path) { ... }`. If the flagged file is an extra, do **not** delete the main film

**Comment by aetaric** [comment]
This is suspected LLM output. As such this issue will be summarily closed.
